src/index.ts

```diff
@@ -151,3 +151,18 @@ async function compileOas(config: Config) {
     doc: openApiObject,
   };
 }
+
+function skipValidation(validatePaths: Array<string> | Array<RegExp>, ctx: koa.Context) {
+  let dontValidate = !validatePaths.some((path: string | RegExp) => {
+    if (isRegexPattern(path)) {
+      return (path as RegExp).test(ctx.path);
+    } else {
+      return ctx.path.startsWith(path as string);
+    }
+  });
+  return dontValidate;
+}
+
+function isRegexPattern(path: string | RegExp) {
+  return typeof path == 'object' ? true : false;
```

Comment:
Can we do `path instanceof RegExp` instead?

Reply:
Good point! I've removed the `isRegexPattern` function altogether and just added `path instanceof RegExp` so that we don't have to type cast in line 158